docs/custom/error-handlers.mdx

```diff
@@ -0,0 +1,150 @@
+---
+id: error_handlers
+title: Custom error handlers
+---
+
+:::caution
+The following features of AMF error handling are still in development:
+- Support for custom error handling in parsing, validation and emission stages
+- Javascript support
+:::
+
+# What's an AMF error handler
```

Comment:
If you use "What is" you need a question mark here.  Would revise to just ## AMF error handlers

Reply:
Also should be a heading 2, so ## AMF error handlers